pvnet/data/datamodule.py

```diff
@@ -93,7 +93,14 @@ def __init__(
         )
 
     def _get_streamed_samples_dataset(self, start_time, end_time) -> Dataset:
-        return PVNetUKRegionalDataset(self.configuration, start_time=start_time, end_time=end_time)
+        if self.configuration.renewable == "pv":
```

Comment:
should we call this uk_pv, just to be really clear

Reply:
Yep that sounds good, we only need two values of this now I think, "uk_pv" and "site" really, perhaps we should make that clearer in documentation, unless the renewable parameter config is used somewhere else